frontend/public/locales/en/public.json

```diff
@@ -7,6 +7,7 @@
   "Confirm": "Confirm",
   "{{description}} for": "{{description}} for",
   "Edit": "Edit",
+  "{{dateFromNow}}": "{{dateFromNow}}",
```

Comment:
We shouldn't ever have a translation key that is only a parameter. It doesn't make sense. Can you revert this change?

I think we need to disable the i18n test on timestamp. We don't expect the moment.js relative times to be psuedolocalized since they're not using i18next.

Reply:
PR to disable i18n test for timestamps: https://github.com/openshift/console/pull/7445